From ab6397247fb19d580f11ef1330c52ab4757c09cf Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Wed, 25 Sep 2019 20:53:14 +0000
Subject: [PATCH] Fix build with autohybris

PKGCONFIG is still needed to find the android headers in the right place.

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>
---
 core/hybris.pro | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/core/hybris.pro b/core/hybris.pro
index 7c377b9..9a68d90 100644
--- a/core/hybris.pro
+++ b/core/hybris.pro
@@ -20,7 +20,8 @@ HEADERS += hybrisadaptor.h
 LIBS += -L/usr/lib -L../datatypes
 
 !contains(CONFIG,binder) {
-    LIBS += -lhybris-common -lhardware
+    CONFIG += link_pkgconfig
+    PKGCONFIG += android-headers libhardware
 }
 
 include(../common-install.pri)
-- 
2.17.0

